File: python-sdk/src/formanceapi/models/shared/accountresponse.py
import dataclasses
from ..shared import accountwithvolumesandbalances as shared_accountwithvolumesandbalances
from dataclasses_json import dataclass_json
from formanceapi import utils


@dataclass_json
@dataclasses.dataclass
class AccountResponse:
    data: shared_accountwithvolumesandbalances.AccountWithVolumesAndBalances = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    

File: ts-sdk/src/sdk/models/shared/accountresponse.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { AccountWithVolumesAndBalances } from "./accountwithvolumesandbalances";


export class AccountResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data" })
  data: AccountWithVolumesAndBalances;
}